### PR TITLE
jupyter on docker

### PR DESCRIPTION
**Pull request by cyrusmvahid**
In order to run workshops and training I need a local docker image with Jupyter support.
I added 
- jupyter to python image for both python 2 and 3 
- exposed port 8888
- added instruction to readme.md

### CODE REVIEW

**Comment by cyrusmvahid** [comment]
adding jupyter to docker for python

**Comment by piiswrong** [comment]
@mli

File: docker/install/python.sh [hunk 1]
```diff
@@ -6,5 +6,5 @@ apt-get update && apt-get install -y python-dev python3-dev
 # the version of the pip shipped with ubuntu may be too lower, install a recent version here
 cd /tmp && wget https://bootstrap.pypa.io/get-pip.py && python3 get-pip.py && python2 get-pip.py
 
-pip2 install nose pylint numpy nose-timer requests
-pip3 install nose pylint numpy nose-timer requests
+pip2 install nose pylint numpy nose-timer request jupyter
```
Comment (mli) [inline docker/install/python.sh]: why change from `request` to `requests`

File: docker/Dockerfiles/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# -*- mode: dockerfile -*-
+# part of the dockerfile to install the python binding
+
+COPY install/python.sh install/
+RUN install/python.sh
+RUN pip3 install jupyter
+ENV PYTHONPATH=/mxnet/python
```
Comment (mli) [inline docker/Dockerfiles/Dockerfile]: this is not a complete dockerfile, why do we need it?

Comment (cyrusmvahid) [inline docker/Dockerfiles/Dockerfile]: I have just added installation of jupyter to the existing dockerfile. Basically I want to be able to have an installed jupyter on after running docker. added the installation to the same file that python was installed

**Comment by mli** [comment]
I'm going to close the PR since there is no activity for more than 2 months. But please feel free to re-open it and update with the master.

**Pull request closed without merging**